packages/cubejs-server-core/core/index.js

```diff
@@ -81,6 +81,7 @@ class CubejsServerCore {
       driverFactory: () => CubejsServerCore.createDriver(options.dbType),
       apiSecret: process.env.CUBEJS_API_SECRET,
       dbType: process.env.CUBEJS_DB_TYPE,
+      devServer: true,
```

Comment:
To ensure backward compatibility should be `devServer: process.env.NODE_ENV !== 'production'`

Reply:
Done.